app/src/main/java/com/eveningoutpost/dexdrip/Services/LibreWifiReader.java

```diff
@@ -453,30 +509,30 @@ private void readData() {
         } else {
             recieversIpAddresses = Pref.getString("wifi_recievers_addresses", "");
         }
-        int packetsToRead = 1;
+        int packetsToRead = isLibre2() ? 5 : 1;
         Log.d(TAG, "reading " + packetsToRead + " packets");
         LibreWifiData[] LibreWifiDataArr = Read(recieversIpAddresses, packetsToRead);
-        
+
         Log.d(TAG, "After reading ..." + LibreWifiDataArr);
 
         if (LibreWifiDataArr == null || LibreWifiDataArr.length == 0) {
             return;
         }
-        Log.d(TAG, "After verification ..." + LibreWifiDataArr);
+        Log.d(TAG, "After verification ... size = " + LibreWifiDataArr.length);
         // Last in the array is the most updated reading we have.
         for (LibreWifiData LastReading : LibreWifiDataArr) {
             // Last in the array is the most updated reading we have.
             //TransmitterRawData LastReading = LastReadingArr[LastReadingArr.length -1];
 
             //if (LastReading.CaptureDateTime > LastReportedReading + 5000) {
             // Make sure we do not report packets from the far future...
-            
-            Log.d(TAG, "checking packet from " +  JoH.dateTimeText(LastReading.CaptureDateTime ));
-            
-            if ((LastReading.CaptureDateTime > LastReportedTime + 4 * 60000) &&
+
+            Log.d(TAG, "checking packet from " + JoH.dateTimeText(LastReading.CaptureDateTime));
+
+            if ((LastReading.CaptureDateTime > LastReportedTime + 55000) &&
```

Comment:
Another value instead of a constant.

Reply:
Again, if we have a reading before a minute, has passed, this is probably something that we already saw, so we want to avoid this.
So, this is actually a minute - some tolerance.